conda/__version__.py

```diff
@@ -1,5 +1,61 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
-from .auxlib.packaging import get_version
+"""Placeholder for the actual version code injected by hatch-vcs.
 
-__version__ = get_version(__file__)
+The logic here is used during development installs only so keep it simple. Since conda
+abides by CEP-8, which outlines using CalVer, our development version is simply:
+    YY.MM.0.devN+HASH[.dirty]
```

Comment:
There is a version scheme config option in setuptools_scm that deals with CalVer: [calver-by-date](https://github.com/pypa/setuptools_scm/blob/fb261332d9b46aa5a258042d85baa5aa7b9f4fa2/src/setuptools_scm/version.py#L350-L365) (it's missing in the docs)

```
[tool.hatch.version.raw-options]
version_scheme = "calver-by-date"
local_scheme = "dirty-tag"
```

I don't think we need the hash in the version

Reply:
Wow, setuptools_scm does all the right things:

| Version | Notes |
|---|---|
| `23.3.1.dev13+g7127cdba2` | development |
| `23.3.1.dev13+g7127cdba2.d20230321` | dirty |
| `23.3.1.dev15+g8aad8962b` | development |